src/track/trackrecord.cpp

```diff
@@ -22,8 +22,7 @@ bool TrackRecord::updateGlobalKey(
         mixxx::track::io::key::ChromaticKey key,
         mixxx::track::io::key::Source keySource) {
     if (key == mixxx::track::io::key::INVALID) {
-        resetKeys();
-        return true;
+        return false;
```

Comment:
Why did you change the behavior for invalid keys? What is the effect of returning `false` instead of silently resetting the key to the default value without failing?

Reply:
Ok, got it from the previous comments ;)